### PR TITLE
Include ambient.d.ts files in adapter packages.

### PR DESCRIPTION
**Pull request by SG60**
Fixes automatic typing for Cloudflare caches and similar stuff.

### Please don't delete this checklist! Before submitting the PR, please make sure you do the following:
- [ ] It's really useful if your PR references an issue where it is discussed ahead of time. In many cases, features are absent for a reason. For large changes, please create an RFC: https://github.com/sveltejs/rfcs
- [x] This message body should clearly illustrate what problems it solves.
- [ ] Ideally, include a test that fails without this PR but passes with it.

### Tests
- [x] Run the tests with `pnpm test` and lint the project with `pnpm lint` and `pnpm check`

### Changesets
- [x] If your PR makes a change that should be noted in one or more packages' changelogs, generate a changeset by running `pnpm changeset` and following the prompts. All changesets should be `patch` until SvelteKit 1.0

### CODE REVIEW

File: packages/adapter-netlify/package.json [hunk 1]
```diff
@@ -19,6 +19,7 @@
 	"main": "index.js",
 	"types": "index.d.ts",
 	"files": [
+		"ambient.d.ts",
```
Comment (Rich-Harris) [inline packages/adapter-netlify/package.json:22]:
```suggestion
```

File: packages/adapter-node/package.json [hunk 2]
```diff
@@ -19,6 +19,7 @@
 	"main": "index.js",
 	"types": "index.d.ts",
 	"files": [
+		"ambient.d.ts",
```
Comment (Rich-Harris) [inline packages/adapter-node/package.json:22]:
```suggestion
```

File: packages/adapter-vercel/package.json [hunk 3]
```diff
@@ -19,6 +19,7 @@
 	"main": "index.js",
 	"types": "index.d.ts",
 	"files": [
+		"ambient.d.ts",
```
Comment (Rich-Harris) [inline packages/adapter-vercel/package.json:22]:
```suggestion
```

File: .changeset/dull-cows-sneeze.md [hunk 4]
```diff
@@ -0,0 +1,9 @@
+---
+'@sveltejs/adapter-cloudflare': patch
+'@sveltejs/adapter-cloudflare-workers': patch
+'@sveltejs/adapter-netlify': patch
+'@sveltejs/adapter-node': patch
+'@sveltejs/adapter-vercel': patch
```
Comment (Rich-Harris) [inline .changeset/dull-cows-sneeze.md:6]:
```suggestion
```

**Review by Rich-Harris — COMMENTED** [review]
thank you — we actually _don't_ want the `ambient.d.ts` files to be included with `adapter-netlify`, `adapter-node` and `adapter-vercel` because they're purely concerned with the adapter innards. but we do want it in the case of `adapter-cloudflare` and `adapter-cloudflare-workers`. the distinction could probably be clearer but for now it's easier to just do this

**Comment by SG60** [comment]
Ah ok! Sorry, didn't realise that. Thanks for making the changes.

**Pull request merged**